### PR TITLE
fix(ios) fix uploading symbols to Crashlytics

### PR DESCRIPTION
**Pull request by saghul**
Since we no longer use bitcode we need to upload them when we build the app.

### CODE REVIEW

File: ios/fastlane/Fastfile [hunk 1]
```diff
@@ -102,24 +102,10 @@ platform :ios do
       uses_non_exempt_encryption: false
     )
 
+    upload_symbols_to_crashlytics
```
Comment (alexbumbu) [inline ios/fastlane/Fastfile:105]: providing the path is not needed?

Comment (saghul) [reply ios/fastlane/Fastfile:105]: It should be automagic, as it was before. I can double check the latest beta, which included this commit.

Comment (alexbumbu) [reply ios/fastlane/Fastfile:105]: I'm curios if it works. How I understood this works is that you're supposed to download the dsym file, then upload it, but in this case I'm not sure it knows where to get the file from without providing the path.

**Comment by saghul** [comment]
Screenshot 2022-11-10 at 10 06 27

AFAICT the dsyms were uploaded successfully.

**Review by alexbumbu — APPROVED** [review]

**Pull request merged**